packages/twenty-server/.env.example

```diff
@@ -55,3 +55,4 @@ SIGN_IN_PREFILLED=true
 # PASSWORD_RESET_TOKEN_EXPIRES_IN=5m
 # API_RATE_LIMITING_TTL=
 # API_RATE_LIMITING_LIMIT=
+# MAXIMUM_RECORD_AFFECTED=100
```

Comment:
Not sure if it should be an env variable or simply a global config (pg_graphql.config.ts?) somewhere in the code? 
I see what you mean and can imagine this value might depend on the environment because of different DB capabilities though 🤔

Reply:
In my concern it's more a security config, so someone deploying the project should easily change this to the value he want.
cc @charlesBochet what do you think about that ?

Reply:
I like the new name: MUTATION_MAXIMUM_RECORD_AFFECTED